app/lib/money/money.ts

```diff
@@ -69,7 +69,9 @@ const currencyDataMap: CurrencyDataMap = {
             decimals: this.decimals,
           });
           formattedValue = formattedValue.replace('BTC', '').trim();
-          return `${this.symbol}${formattedValue}`;
+          return options.currency
```

Comment:
bug, formatting BTC was not excluding the the symbol with `money.toLocaleString({ showCurrency: false })`